tests/resourcemanager/src/main.cpp

```diff
@@ -0,0 +1,68 @@
+/*
```

Comment:
Test should be removed I guess. It is tested in lib.

Reply:
In aos core lib JSON parsing for resource config is not covered.

These test cases check zephyr json parsing of the resource config, for example.
I'd keep these tests as a separate target.